platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/AbstractTransformExecution.java

```diff
@@ -291,6 +294,10 @@ public void markLegacySnapshottingInputsFinished(CachingState cachingState) {
 
     @Override
     public Optional<CachingDisabledReason> shouldDisableCaching(@Nullable OverlappingOutputs detectedOverlappingOutputs) {
+        if (System.getProperty(CACHING_DISABLED_PROPERTY) != null) {
```

Comment:
❓ Should we also check if value is true or false?

Reply:
I had that at first, but instead I decided to make it a no-value property.
Usage is `./gradlew -Dorg.gradle.experimental.transform-caching-disabled task`.